plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/AbstractPluginTest.kt

```diff
@@ -3,18 +3,17 @@ package org.jlleitschuh.gradle.ktlint
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.GradleRunner
 import org.gradle.util.TextUtil.normaliseFileSeparators
-import org.junit.Rule
-import org.junit.rules.TemporaryFolder
+import org.junit.jupiter.api.io.TempDir
 import java.io.File
 import java.util.Properties
 
 abstract class AbstractPluginTest {
 
-    @get:Rule
-    val temporaryFolder = TemporaryFolder()
+    @TempDir
+    lateinit var temporaryFolder: File
```

Comment:
Cool! When did they add this to the official API? I thought it was just part of [junit-pioneer](https://github.com/junit-pioneer/junit-pioneer)!

Reply:
In latest `5.4` release: https://junit.org/junit5/docs/snapshot/release-notes/index.html#release-notes-5.4.0